index.js

```diff
@@ -72,10 +72,12 @@ Options:
         console.error(`  ${red('Remember: ')} If you have some money to spare, donate it to twist.moe so they can the servers up and running!`)
         try{
             const donation = await getJSON('/api/donation')
-            console.error(`\n  They're ${red(donation.remaining + '$ short')} on money this month.\n  To donate, please visit https://twist.moe/\n`)
+            console.error(`\n  They're ${red(Math.round((donation.remaining + Number.EPSILON)*100)/100 + '$ short')} on money this month.\n  To donate, please visit https://twist.moe/\n`)
         }catch(err){}
         console.error(`  ${yellow(getTitle(selectedAnime))}\n`)
-
+        if(!(fs.existsSync(path.resolve(process.cwd(), argv.output || '')))){
+            fs.mkdirSync(path.resolve(process.cwd(), argv.output || ''));
+        }
```

Comment:
```suggestion
            fs.mkdirSync(path.resolve(process.cwd(), argv.output || ''), { recursive: true });
```
This will create parent directories as well and won't throw exceptions if the folder already exists. With this it shouldn't be necessary to check whether the directory exists.

Reply:
looks good to me